core/arch/arm/plat-imx/registers/imx-regs.h

```diff
@@ -0,0 +1,42 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2015 Freescale Semiconductor, Inc.
+ * All rights reserved.
+ * Copyright (c) 2016, Wind River Systems.
+ * All rights reserved.
```

Comment:
Please consider the windriver copyright.

Reply:
Not sure to understand. 
i kept the windriver copyright because this file was move from plat-imx to register. So i kept the copyright.
do you want me to remove it?

Reply:
The initial header file (**core/arch/arm/plat-imx/imx-regs.h**) initially detailed the license content:
```
 * Redistribution and use in source and binary forms, with or without
 * modification, are permitted provided that the following conditions are met:
 *
 * 1. Redistributions of source code must retain the above copyright notice,
 * this list of conditions and the following disclaimer.
 *
 * 2. Redistributions in binary form must reproduce the above copyright notice,
 * this list of conditions and the following disclaimer in the documentation
 * and/or other materials provided with the distribution.
 *
 * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
 * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
 * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
 * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
 * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
 * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
 * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
 * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
 * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
 * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 * POSSIBILITY OF SUCH DAMAGE.
```
Please preserve this content. It seems useless but you have to.

Reply:
ok sure

Reply:
@etienne-lms , i spoke too fast, 
actually i did not remove it. it was already done and we were able to do so because it is oly our own (i.e NXP) copyright file.
i have restored the license content to the other files that we share the copyright with windriver.
is that ok withyou?

Reply:
There is this Wind River System copyright that makes me think you don't own the full file content, hence cannot change the license info. If you rewrite the file from scratch, then it is fine to apply only your copyright and choose the license info you wish.

Reply:
ok i got it, yes indeed the imx-regs.h need the license content. i will add it back to the file

Reply:
comment thread addressed